app/src/redux/reducer.js

```diff
@@ -10,6 +10,9 @@ import newEvent from './newEventReducers';
 import editEvent from './editEventReducers';
 import contactInfo from './contactInfo';
 import gift from './gift';
+import commentsList from './commentsFetchReducers';
```

Comment:
create one reducer and check how it should looks in Denys PR (action and reducer is separated )